sdk/ml/azure-ai-ml/azure/ai/ml/operations/_registry_operations.py

```diff
@@ -87,11 +92,41 @@ def _check_registry_name(self, name) -> str:
             )
         return registry_name
 
+    def _get_polling(self, name):
+        """Return the polling with custom poll interval."""
+        path_format_arguments = {
+            "registryName": name,
+            "resourceGroupName": self._resource_group_name,
+        }
+        return AzureMLPolling(
+            LROConfigurations.POLL_INTERVAL,
+            path_format_arguments=path_format_arguments,
+        )
+
     @monitor_with_activity(logger, "Registry.BeginCreate", ActivityType.PUBLICAPI)
-    def begin_create_or_update(
+    def begin_create(
         self,
         registry: Registry,
-        update_dependent_resources: bool = False,
         **kwargs: Dict,
-    ) -> LROPoller[RestRegistry]:
-        raise NotImplementedError("Placeholder until implemented in subsequent task.")
+    ) -> LROPoller["_models.Registry"]:
+        """Create a new Azure Machine Learning Registry.
+
+        Returns the registry if already exists.
+
+        :param registry: Registry definition.
+        :type registry: Registry
+        :type update_dependent_resources: boolean
```

Comment:
could not find "update_dependent_resources"?

Reply:
Deleted unused param.